app/components/DeleteTodo.js

```diff
@@ -10,6 +10,12 @@ class DeleteTodo extends Component {
         this.state = {}
     }
 
+    function update() {
```

Comment:
The function name "update" is ambiguous and may cause confusion. Consider renaming it to something more descriptive like "handleDelete".